src/common/transformations/src/transformations/common_optimizations/fuse_rotary_positional_embeddings.cpp

```diff
@@ -165,92 +165,6 @@ ov::pass::RoPEFusionFlux::RoPEFusionFlux() {
-                                             size_t axis) {
-    auto slice_start = ParseSymbolVariant({start});
-    auto slice_stop = ParseSymbolVariant({stop});
-    auto slice_step = ParseSymbolVariant({step});
-    auto slice_axis = ParseSymbolVariant({static_cast<int64_t>(axis)});
-
-    auto opt1 = pattern::wrap_type<ov::opset8::Slice>({data, slice_start, slice_stop, slice_step, slice_axis});
-
-    std::vector<symbol_variant> vbegin(axis + 1, 0);
-    std::vector<symbol_variant> vend(axis + 1, 0);
-    std::vector<symbol_variant> vstride(axis + 1, 1);
-
-    vbegin[axis] = start;
-    vend[axis] = stop;
-    vstride[axis] = step;
-
-    auto begin = ParseSymbolVariant(vbegin);
-    auto end = ParseSymbolVariant(vend);
-    auto stride = ParseSymbolVariant(vstride);
-
-    std::vector<int64_t> begin_mask(axis + 1, 1);
-    std::vector<int64_t> end_mask(axis + 1, 1);
-    std::vector<int64_t> new_axis_mask;
-    std::vector<int64_t> shrink_axis_mask;
-    std::vector<int64_t> ellipsis_mask;
-
-    begin_mask[axis] = 0;
-    end_mask[axis] = 0;
-
-    auto opt2 = pattern::wrap_type<ov::op::v1::StridedSlice>({data, begin, end, stride},
-                                                             {{"begin_mask", begin_mask},
-                                                              {"end_mask", end_mask},
-                                                              {"new_axis_mask", new_axis_mask},
-                                                              {"shrink_axis_mask", shrink_axis_mask},
-                                                              {"ellipsis_mask", ellipsis_mask}});
-
-    return opt1 | opt2;
-}
-
-static std::shared_ptr<ov::Node> NewGenStridedSlice(std::shared_ptr<ov::Node> data,
-                                                    const pattern::PatternOp& start,
-                                                    const pattern::PatternOp& stop,
-                                                    const pattern::PatternOp& step,
-                                                    size_t axis) {
-    std::vector<int64_t> begin_mask(axis + 1, 1);
-    std::vector<int64_t> end_mask(axis + 1, 1);
-    std::vector<int64_t> new_axis_mask;
-    std::vector<int64_t> shrink_axis_mask;
-    std::vector<int64_t> ellipsis_mask;
-
-    begin_mask[axis] = 0;
-    end_mask[axis] = 0;
-
-    return pattern::wrap_type<ov::op::v1::StridedSlice>({data, start, stop, step},
-                                                        {{"begin_mask", begin_mask},
-                                                         {"end_mask", end_mask},
-                                                         {"new_axis_mask", new_axis_mask},
-                                                         {"shrink_axis_mask", shrink_axis_mask},
-                                                         {"ellipsis_mask", ellipsis_mask}});
-}
 
 ov::pass::RoPEFusionGPTNEOX::RoPEFusionGPTNEOX() {
     using namespace ov::op;
```

Comment:
```suggestion
    using namespace ov::op;
    using ov::op::util;
```
And please remove other changes in this transformation